### PR TITLE
Page b2b

### PR DESCRIPTION
**Pull request by cedricmenteau**
@ssaunier Pour review!

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -290,6 +290,22 @@ en:
     shop:
       title: "Le Wagon's shop"
       description: "Because coders got style"
+    enterprise:
+      title: "Boost you team and gain agility"
```
Comment (krokrob) [inline config/locales/en.yml]: Boost **your** team

File: config/locales/fr.yml [hunk 2]
```diff
@@ -283,6 +283,22 @@ fr:
...
       title: "La boutique du Wagon"
       description: "Le nouveau chic - Paris"
+    enterprise:
+      title: "Boostez vos équipes et gagnez en agilité"
+      catchline: "Apprenez à coder et faites la différence dans un monde en profondes mutations"
+      cta_banner: "Discutons-en !"
+      usecase_1: "Vous avez besoin de recruter des profils techniques"
+      usecase_2: "Vous souhaitez freiner le turnover de vos collaborateurs"
+      usecase_3: "Vous voulez améliorer la qualité des échanges entre les équipes techniques et les chefs de projet"
+      usecase_4: "Vous voulez faire de vos chefs de projet des “product owner"
```
Comment (krokrob) [inline config/locales/fr.yml]: **owners** no?

File: config/locales/fr.yml [hunk 3]
```diff
@@ -283,6 +283,22 @@ fr:
...
+      usecase_5: "Vous avez besoin d’innover, pour attirer des jeunes profils"
+      usecase_6: "Vous voulez réduire le time to market grace aux méthodologie de lancement d’un MVP"
```
Comment (krokrob) [inline config/locales/fr.yml]: grâce

Comment (krokrob) [inline config/locales/fr.yml]: méthodologies

File: app/views/pages/enterprise.html.erb [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      <%= link_to 'https://lewagon.typeform.com/to/LOs0uR', target: '_blank', class: 'btn banner-cta-item btn-danger btn-lg' do %>
+        <%= t(".cta_contact") %>
+      <% end %>
+    </div>
+    <div class="contact-details">
+      <i>ou</i>
+      <a href="tel:<%= t(".phone_number").gsub(' ', '') %>"><%= t(".phone_number") %></a>
+      <a href="mailto:<%= t(".mail_adress") %>?Subject=Contact%20B2B"><%= t(".mail_adress") %></a>
+  </div>
+</div>
```
Comment (krokrob) [inline app/views/pages/enterprise.html.erb]: you can remove useless line breaks and add one at final line

File: config/locales/fr.yml [hunk 5]
```diff
@@ -283,6 +283,22 @@ fr:
...
+      cta_usecases: Découvrez nos formations
+      quote_content: "Learning to code today is like learning to read, without it the future will be close."
+      quote_author: "Bronwen Valve"
+      cta_contact: "Échangez avec Benjamin !"
+      phone_number: "+33 (0)7 83 59 97 95"
+      mail_adress: "benjamin@lewagon.com"
```
Comment (krokrob) [inline config/locales/fr.yml]: .org

**Comment by krokrob** [comment]
Maybe something is missing but the layout is broken
capture d ecran 2017-03-17 a 16 56 39

File: config/locales/en.yml [hunk 6]
```diff
@@ -290,6 +290,22 @@ en:
...
+      catchline: "Learn to code and make the difference in a world in constant change"
+      cta_banner: "I want to know more"
+      usecase_1: "Need to recruit technical profiles?"
+      usecase_2: "You want to keep your most promising employees?"
+      usecase_3: "You want to improve the quality of the exchanges between the technical teams and the project managers?"
+      usecase_4: "You want to transform your project managers into product owners?"
+      usecase_5: "You need to innovate, to attract young profiles?"
+      usecase_6: "You want to reduce the time to market by learning how to launch an MVP?"
+      cta_usecases: Learn about our trainings
+      quote_content: "Learning to code today is like learning to read, without it the future will be close."
```
Comment (Eschults) [inline config/locales/en.yml]: close**d**

File: config/locales/fr.yml [hunk 7]
```diff
@@ -283,6 +283,22 @@ fr:
...
+      catchline: "Apprenez à coder et faites la différence dans un monde en profondes mutations"
+      cta_banner: "Discutons-en !"
+      usecase_1: "Vous avez besoin de recruter des profils techniques"
+      usecase_2: "Vous souhaitez freiner le turnover de vos collaborateurs"
+      usecase_3: "Vous voulez améliorer la qualité des échanges entre les équipes techniques et les chefs de projet"
+      usecase_4: "Vous voulez faire de vos chefs de projet des “product owner"
+      usecase_5: "Vous avez besoin d’innover, pour attirer des jeunes profils"
+      usecase_6: "Vous voulez réduire le time to market grace aux méthodologie de lancement d’un MVP"
+      cta_usecases: Découvrez nos formations
+      quote_content: "Learning to code today is like learning to read, without it the future will be close."
```
Comment (Eschults) [inline config/locales/fr.yml]: close**d**

File: config/locales/pt-BR.yml [hunk 8]
```diff
@@ -296,6 +296,22 @@ pt-BR:
...
+      catchline: "Learn to code and make the difference in a world in constant change"
+      cta_banner: "I want to know more"
+      usecase_1: "Need to recruit technical profiles?"
+      usecase_2: "You want to keep your most promising employees?"
+      usecase_3: "You want to improve the quality of the exchanges between the technical teams and the project managers?"
+      usecase_4: "You want to transform your project managers into product owners?"
+      usecase_5: "You need to innovate, to attract young profiles?"
+      usecase_6: "You want to reduce the time to market by learning how to launch an MVP?"
+      cta_usecases: Learn about our trainings
+      quote_content: "Learning to code today is like learning to read, without it the future will be close."
```
Comment (Eschults) [inline config/locales/pt-BR.yml]: close**d**

**Pull request closed without merging**